content/docs/2.14/reference/_index.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Reference"
+weight = 2
++++
+
+Reference information for the KEDA autoscaler.
+
+- [Scaled Object specification](./scaledobject-spec)
+- [ScaledJob specification](./scaledjob-spec.md)
+- [Events](./events)
```

Comment:
Shouldn't you add .md at the end?

Reply:
You would think so, but I investigated this earlier and it seems as if the site generator handles this automatically for relative paths. I keep missing instances (like the above). File extensions are so common that they're almost invisible ...

Reply:
According to the build log, https://app.netlify.com/sites/keda/deploys/667af00917bae9000859d0c6, the link checker seems to not care much between .md and not. IIRC, Hugo prefers .md not be present for internal links.